packages/core/src/oidc/init.ts

```diff
@@ -210,98 +206,31 @@ export default function initOidc(
       },
     },
     extraParams: Object.values(ExtraParamsKey),
-    // eslint-disable-next-line complexity
+
     extraTokenClaims: async (ctx, token) => {
-      const { isDevFeaturesEnabled, isCloud } = EnvSet.values;
+      const organizationApiResourceClaims = await getExtraTokenClaimsForOrganizationApiResource(
+        ctx,
+        token
+      );
+
+      const jwtCustomizedClaims = await getExtraTokenClaimsForJwtCustomization(
```

Comment:
Use object?

Reply:
Could you please be more specific?

Reply:
There are more than 5 input parameters for that function, should replace them using a parameters object?

Reply:
updated